cmake/stm32/common.cmake

```
@@ -0,0 +1,162 @@
+set(STM32_SUPPORTED_FAMILIES L0 L1 L4 F0 F1 F2 F3 F4 G4 L4 F7 H7)

```

G0 is supported too :)
```suggestion
set(STM32_SUPPORTED_FAMILIES F0 F1 F2 F3 F4 F7 G0 G4 H7 L0 L1 L4)

```